ceph-releases/kraken/ubuntu/16.04/demo/entrypoint.sh

```diff
@@ -215,6 +215,12 @@ function bootstrap_rgw {
     mkdir -p /var/lib/ceph/radosgw/${RGW_NAME}
     ceph ${CEPH_OPTS} auth get-or-create client.radosgw.gateway osd 'allow rwx' mon 'allow rw' -o /var/lib/ceph/radosgw/${RGW_NAME}/keyring
     chown -R ceph. /var/lib/ceph/radosgw/${RGW_NAME}
+
+    #configure rgw dns name
+    cat <<ENDHERE >>/etc/ceph/${CLUSTER}.conf
```

Comment:
Just in case we restart the container, please check if section exists by doing:

`if [[ ! "$(egrep "\[client.radosgw.gateway\]" /etc/ceph/${CLUSTER}.conf)" ]]; then`